assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -66,6 +69,9 @@ type FormState = {
   thankYouPageStage: ThankYouPageStage,
   hasSeenDirectDebitThankYouPageCopy: boolean,
   payPalHasLoaded: boolean,
+  isSignInRequired: boolean,
```

Comment:
tiny naming preference (feel free to ignore) but I think these sort of variables are better named without the `is` prefix, which I don't think adds anything